applications/luci-app-transmission/htdocs/luci-static/resources/view/transmission.js

```diff
@@ -0,0 +1,273 @@
+		setFlagBool(o);
+
+		o = s.option(form.Value, 'incomplete_dir', _('Incomplete directory'));
+		o.depends('incomplete_dir_enabled', 'true');
+
+		o = s.option(form.ListValue, 'preallocation', _('Preallocation'));
+		o.value('0', _('Off'));
+		o.value('1', _('Fast'));
+		o.value('2', _('Full'));
+
+		o = s.option(form.Flag, 'rename_partial_files', _('Rename partial files'));
+		setFlagBool(o);
+
+		o = s.option(form.Flag, 'start_added_torrents', _('Automatically start added torrents'));
+		setFlagBool(o);
+
+		o = s.option(form.Flag, 'trash_original_torrent_files', _('Trash original torrent files'));
+		setFlagBool(o);
+
+		o = s.option(form.Value, 'umask', 'umask');
+
+		o = s.option(form.Flag, 'watch_dir_enabled', _('Enable watch directory'));
+		setFlagBool(o);
+
+		o = s.option(form.Value, 'watch_dir', _('Watch directory'));
+		o.depends('watch_dir_enabled', 'true');
+
+
+		s = m.section(form.TypedSection, 'transmission', _('Miscellaneous'));
+		s.anonymous = true;
+
+		o = s.option(form.Value, 'cache_size_mb', _('Cache size in MB'));
+
+		o = s.option(form.Flag, 'dht_enabled', _('DHT enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.ListValue, 'encryption', _('Encryption'));
+		o.value('0', _('Prefer unencrypted'));
+		o.value('1', _('Prefer encrypted'));
+		o.value('2', _('Require encrypted'));
+
+		o = s.option(form.Flag, 'lazy_bitfield_enabled', _('Lazy bitfield enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.Flag, 'lpd_enabled', _('LPD enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.ListValue, 'message_level', _('Message level'));
+		o.value('0', _('None'));
+		o.value('1', _('Error'));
+		o.value('2', _('Info'));
+		o.value('3', _('Debug'));
+
+		o = s.option(form.Flag, 'pex_enabled', _('PEX enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.Flag, 'prefetch_enabled', _('Prefetch enabled'));
+		// This defaults to false if configured with --enable-lightweight.
+
+		o = s.option(form.Flag, 'scrape_paused_torrents_enabled', _('Scrape paused torrents enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.Flag, 'script_torrent_done_enabled', _('Script torrent done enabled'));
+		setFlagBool(o);
+
+		o = s.option(form.Value, 'script_torrent_done_filename', _('Script torrent done filename'));
+		o.depends('script_torrent_done_enabled', 'true');
+
+		o = s.option(form.Flag, 'utp_enabled', _('uTP enabled'));
+		setFlagBool(o);
+
+
+		s = m.section(form.TypedSection, 'transmission', _('Peer settings'));
+		s.anonymous = true;
+
+		o = s.option(form.Value, 'bind_address_ipv4', _('Binding address IPv4'));
+
+		o = s.option(form.Value, 'bind_address_ipv6', _('Binding address IPv6'));
+
+		o = s.option(form.Value, 'peer_congestion_algorithm', _('Peer congestion algorithm'),
+			_('This is documented on <a href="https://www.irif.fr/~jch/software/bittorrent/tcp-congestion-control.html" target="_blank" rel="noreferrer noopener">tcp-congestion-control</a>.'));
```

Comment:
Transmission [wiki](https://github.com/transmission/transmission/wiki/Editing-Configuration-Files#peers) provides a dead [link](http://www.pps.jussieu.fr/~jch/software/bittorrent/tcp-congestion-control.html). I replaced with this working link.